app/scripts/modules/core/delivery/executions/executions.less

```diff
@@ -75,7 +75,10 @@ executions {
         margin-top: 0;
         font-weight: 600;
       }
-      &.group-by-name {
```

Comment:
I guess there's no group by name? It almost feels like it shouldn't matter which `group-by-*`, `.account-tag` should also be `display: inline`, so maybe just remove the `.group-by-*`? (FWIW, group by none is also broken...)